packages/web-components/src/pane/pane.ts

```diff
@@ -0,0 +1,260 @@
+import { attr, FASTElement, observable, Updates } from '@microsoft/fast-element';
+import { isHTMLElement, keyEscape, keyTab } from '@microsoft/fast-web-utilities';
+import { PanePosition, PaneSize } from './pane.options.js';
+
+export interface OpenEvent {
+  open: boolean;
+  position?: string;
+  controlSize?: PaneSize | number;
+}
+
+// The Pane class extends the FASTElement which provides a foundation for creating
+// custom elements using FAST. It extends the HTMLElement interface of the browser.
+export class Pane extends FASTElement {
+  private panes: Pane[] = [];
+
+  public connectedCallback(): void {
+    super.connectedCallback();
+    // Register event listener
+    this.addEventListener('keydown', this.handleKeyDown);
+    this.setItems();
+  }
+
+  public disconnectedCallback(): void {
+    super.disconnectedCallback();
+    // Unregister event listener
+    this.removeEventListener('keydown', this.handleKeyDown);
+  }
+
+  private hasInteracted: boolean = false;
+
+  // The previous active element before opening the pane.
+  private previousActiveElement?: HTMLElement;
+
+  /**
+   * The pane element.
+   * @public
+   * @remarks
+   * HTML Attribute: pane
+   */
+  @observable
+  public _pane?: HTMLElement;
```

Comment:
There should be parity between the types for _pane and panes

Reply:
Yeah, if Pane has a type should it not be used here?